mobile/src/calendar/event-dialog/edit-sick-leave-event-dialog.tsx

```diff
@@ -31,14 +33,18 @@ class EditSickLeaveEventDialogImpl extends Component<EditSickLeaveEventDialogPro
     public render() {
         const selectedSickLeave = this.selectedSickLeave();
         const isEditable = selectedSickLeave ? !selectedSickLeave.calendarEvent.isCompleted : false;
+        let isDisabledForComplete = true;
+        if (this.props.completeSickLeavePermission === true && !isEditable) {
+            isDisabledForComplete = false;
+        }
```

Comment:
This code can be simplified with ternary operator or even just to something like
```
const isDisabledForComplete = !(this.props.completeSickLeavePermission === true && !isEditable)
```

Reply:
@YaroTs IMO, it doesn't look simple %)